build-config/make/grub_aarch64.make

```diff
@@ -0,0 +1,223 @@
+#-------------------------------------------------------------------------------
```

Comment:
how is this file different from grub.make?  Having this copy will be a huge pain to maintain.  It will be better if we can keep everything in grub.make and work out any differences there.